asm/patches/instant_ko.asm

```diff
@@ -0,0 +1,85 @@
+.org @NextFreeSpace
```

Comment:
`.open "sys/main.dol"` Needs this at the top of the file, then rerun `assemble.py` in the ASM Folder.

Reply:
@wooferzfg I did the above in my local machine and it fixed the issue. I'll keep an eye out for the eye when it's built, but I've tested it with this fix locally, and all the instant kills worked.

The best was you can kill yourself with soup, and if you Save and Quit then you start the file with double damage and 3 hearts, I'm going to leave that in as a fun little bonus.

Reply:
@Crain-32 Could you open a new PR based on my branch? That way you can make your own changes again